code/game/machinery/autolathe.dm

```diff
@@ -365,7 +365,7 @@
 
 /obj/machinery/autolathe/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
 	. = ..()
-	if((!HAS_SILICON_ACCESS(usr) && !isAdminGhostAI(usr)) && !Adjacent(usr))
+	if(isobserver(usr) || (!HAS_SILICON_ACCESS(usr) && !isAdminGhostAI(usr)) && !Adjacent(usr))
```

Comment:
If this works for observers, then it probably works for people who are stunned, crit, or dead. It should probably be performing some `can_interact` or `can_perform_action` check`.

Reply:
Trying out can_interact does not work. Not sure why i dont have AI_Interact enabled .